inference-engine/src/multi_device/multi_device_exec_network.hpp

```diff
@@ -163,22 +189,21 @@ class MultiDeviceExecutableNetwork : public InferenceEngine::ExecutableNetworkTh
     static bool RunPipelineTask(InferenceEngine::Task& inferPipelineTask,
                                 NotBusyWorkerRequests& idleWorkerRequests,
                                 const DeviceName& preferred_device);
+    void TryToLoadNetWork(AutoLoadContext& context,
+                          const std::string& modelPath,
+                          const InferenceEngine::CNNNetwork& network);
 
 private:
     std::shared_ptr<InferenceEngine::ICore>                             _core;
     InferenceEngine::IStreamsExecutor::Ptr                              _executor;
     MultiDeviceInferencePlugin*                                         _multiPlugin;
-    InferenceEngine::SoExecutableNetworkInternal                        _networkFirstReady;
```

Comment:
Recently merged PR #8389 uses this class member. Please rebase and fix build issues

Reply:
rebased the code